chain/bitcoind_conn.go

```diff
@@ -471,6 +660,101 @@ func (c *BitcoindConn) txEventHandler() {
 	}
 }
 
+// txEventHandlerRPC is a goroutine that uses the RPC client to check the
+// mempool for new transactions.
+func (c *BitcoindConn) txEventHandlerRPC() {
```

Comment:
same nits from above func re ticker.Stop, time.Sleep and var variable apply below